prov/shm/src/smr_ep.c

```diff
@@ -635,8 +635,16 @@ static int smr_format_sar(struct smr_ep *ep, struct smr_cmd *cmd,
 }
 
 int smr_select_proto(enum fi_hmem_iface iface, bool use_ipc, bool cma_avail,
-                     uint32_t op, uint64_t total_len, uint64_t op_flags)
+                     bool gdrcopy_avail, uint32_t op, uint64_t total_len,
+                     uint64_t op_flags)
 {
+	if (gdrcopy_avail) {
```

Comment:
I think this can be reformatted as

```
if (gdrcopy_avail and total_len <= smr_env.max_gdrcopy_size)
    return total_len <= SMR_MSG_DATA_LEN? smr_src_inline : smr_src_inject;
``` 

so gdrcopy related variable can be put in the same line.

Reply:
Cool thanks! Will update.